### PR TITLE
Checkstyle for src

### PR DESCRIPTION
**Pull request by kalaspuffar**
Hi @bertfrees and @PaulRambags

This is the seconds batch of checkstyle fixes. I've used the tools in IntelliJ to:

* Organize imports
* Format file
* Tabs to spaces
* Adding punctuations to comments so they could be used correctly in javadoc
* Renaming constants to UPPERCASE_UPPERCASE style
* Renaming member variables from constant form camelCase
* Fixing length of strings to below 120 chars
* Changed println and printStacktrace statements to use a logger instead.
* Corrected one FIXME as the solution was to remove that deprecated constructor.
* Renamed variables, files, and directories that had _. Example 
** sv_SE -> sv
** Braillo200_270_400_v1_11Embosser -> Braillo200x270x400v1x11Embosser

There is one file that I keep long lines and suppress warnings because that file was generated.

Best regards
Daniel

### CODE REVIEW

**Review by PaulRambags — APPROVED** [review]

**Comment by bertfrees** [comment]
You did some other things besides formatting and rearranging. Could you summarize everything you did?

**Comment by kalaspuffar** [comment]
Hi @bertfrees 

I think you need to enlighten me about what kind of changes you talking about. To my knowledge, I have told you what changes I did. I had to do some changes for line lengths as mentioned above.

Other changes have been made in other repositories.

Best regards
Daniel

**Comment by bertfrees** [comment]
> Also renamed some variables so they follow rules for members and constants.

OK thanks. I also noticed some classes/files were renamed (you also fixed a FIXME), some comments were removed, some logger calls were added. In the end I skimmed through the whole diff because I discovered some things and wondered what else was hidden in there.

Most of these changes are of course fine (and positive), I just think it's good the mention them (also in the commit message).

The only thing I'm not so sure about is removing comments. Maybe they are wrong, or not useful, but I'm not sure that's the case here. I guess they just don't belong in a commit that is about style/formatting fixes.

**Comment by kalaspuffar** [comment]
Hi @bertfrees 

I think the few cases where I removed code looked like debugging code with println statements in it. To allow it with the new rules I had to change to log statements.

Best regards
Daniel

**Review by bertfrees — APPROVED** [review]

**Comment by kalaspuffar** [comment]
PR approved, will run regression tests.

**Comment by kalaspuffar** [comment]
Regression tests OK, merging.
